lib/account/models/profile_feature.dart

```diff
@@ -137,4 +143,31 @@ extension FeatureExtension on Feature {
         return S.of(context).modelProfileFeatureRequires3G;
     }
   }
+
+  bool isMutuallyExclusive(Feature other) {
+    switch (this) {
+      case Feature.noSmoking:
+        return other == Feature.smoking;
+      case Feature.smoking:
+        return other == Feature.noSmoking;
+      case Feature.noVaping:
+        return other == Feature.vaping;
+      case Feature.vaping:
+        return other == Feature.noVaping;
+      case Feature.noPetsAllowed:
+        return other == Feature.petsAllowed;
+      case Feature.petsAllowed:
+        return other == Feature.noPetsAllowed;
+      case Feature.noChildrenAllowed:
+        return other == Feature.childrenAllowed;
+      case Feature.childrenAllowed:
+        return other == Feature.noChildrenAllowed;
+      case Feature.relaxedDrivingStyle:
+        return other == Feature.speedyDrivingStyle;
+      case Feature.speedyDrivingStyle:
+        return other == Feature.relaxedDrivingStyle;
```

Comment:
Maybe also talkative and quiet?

Reply:
Yes please! I also argued with Fynn about that

Reply:
Lol yeah I know but we agreed to only forbid exact opposites. We are not the profile feature police. If someone wants to talk half the journey and be quiet the other half, who are we to stand in the way of that?

Reply:
but if someone does both wouldn't they just not mark any of the features. I understand talkative as if you can, you should talk and quiet means if you can please stay quiet. If you do not perfere one or the other then don't mark it as a feature?

Reply:
But that gets us in edge cases real fast. Should "talkative" and "music" also be exclusive? You can't do both at the same time either

Reply:
Maybe Fynn is Right. I mean in the End it is not our problem.

Reply:
sure, It does not hurt to leave it in atm.